modules/navier_stokes/src/correctors/NSPressurePin.C

```diff
@@ -48,6 +48,7 @@ NSPressurePin::validParams()
       "pressure_average", "A postprocessor that computes the average of the pressure variable");
 
   params.addClassDescription("Pins the pressure after a solve");
+  params.registerBase("Corrector");
```

Comment:
Just curious, is this now necessary for every corrector class?

Reply:
until we make a base class and decide what goes in it, yes